### PR TITLE
feat!: Meteor 3.0 - Node.js 20

### PR DESCRIPTION
**Pull request by ggazzo**

### CODE REVIEW

File: apps/meteor/.docker/Dockerfile.alpine [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM node:14.21.3-alpine3.16
+FROM node:20.9.0-alpine3.16
```
Comment (ggazzo) [inline apps/meteor/.docker/Dockerfile.alpine:1]:
```suggestion
FROM node:20.9.0-alpine3.18
```

**Review by sampaiodiego — DISMISSED** [review]

**Comment by KevLehman** [comment]
I vote for merging this into V7 so we start using it on development and fix issues as they appear, if any 😬

**Comment by ggazzo** [comment]
waiting for reviews :p 
image

**Review by KevLehman — APPROVED** [review]
We ball

**Review by KevLehman — DISMISSED** [review]

**Review by sampaiodiego — APPROVED** [review]

**Review by KevLehman — DISMISSED** [review]

**Review by sampaiodiego — APPROVED** [review]

**Review by sampaiodiego — APPROVED** [review]

**Review by KevLehman — APPROVED** [review]

**Pull request merged**